packages/editor/src/client/atoms/emojiButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {forwardRef, ButtonHTMLAttributes, useState, useRef} from 'react'
+import {useSlate} from 'slate-react'
+
+import {Icon, Popover, Whisper} from 'rsuite'
+import {SVGIcon} from 'rsuite/lib/@types/common'
+import {IconNames} from 'rsuite/lib/Icon/Icon'
+
+import './emojiButton.less'
+import {Picker, BaseEmoji} from 'emoji-mart'
+
+export interface EmojiButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  readonly icon: IconNames | SVGIcon
+  readonly iconActive?: IconNames | SVGIcon
+  readonly active?: boolean
+}
+
+export const BaseEmojiButton = forwardRef<HTMLButtonElement, EmojiButtonProps>(
+  function BaseEmojiButton({icon, iconActive, active, ...props}, ref) {
+    icon = iconActive && active ? iconActive : icon
+    return (
+      <button
+        style={{
```

Comment:
Is there also a possibility to style this within a css-file instead uf styling it directly in the code?

Reply:
We want to avoid css files as much as possible. @faebebin already opened a jira issue that addresses this problem.

Reply:
This is actually something we might have to discuss  regarding conventions. It could easily be moved to the already existing .lee file. But the .less file here is only for overriding npm-package stylesheet. The other /atoms components use inline css mostly. So where should /atoms/components style live?

Reply:
@faebebin leave it as it is for the moment. We need to clean up the styles anyway and then we can implement a proper style solution that suites all our usecases.

Reply:
ok